README.md

```diff
@@ -55,7 +58,7 @@ Now you can talk to your bot in your mattermost client!
 
 A chat bot is meaningless unless you can extend/customize it to fit your own use cases.
 
-To write a new plugin, simply create a function decorated by `mattermost_bot.bot.respond_to` or `mattermost_bot.bot.listen_to`:
+To write a new plugin, simply create a python file with function decorated by `mattermost_bot.bot.respond_to` or `mattermost_bot.bot.listen_to in the ".../mattermost_bot/plugins" directory `:
```

Comment:
`mattermost_bot.bot.respond_to` or `mattermost_bot.bot.listen_to` in the ".../mattermost_bot/plugins" directory. 
But why in the repo? This battery will be installed as site-packages and you plugins is another project.